index.html

```diff
@@ -80,49 +85,14 @@
     }'>
   </div>
 
-  <div class='mu-erd'
-    data-entities='{
-      "Entity_1": {
-        "ent1_id": {
-          "type": "Integer",
-          "pk": true
-        },
-        "ent2_id": {
-          "type": "Integer",
-          "pk": true,
-          "fk": {
-            "to": { "entity": "Entity_2", "column": "ent2_id" },
-            "type": "one_to_one"
-          }
-        },
-        "ent1_description": {
-          "type": "Varchar"
-        }
-      },
-      "Entity_2": {
-        "ent2_id": {
-          "type": "Integer",
-          "pk": true
-        }
-      },
-      "Entity_3": {
-        "ent3_id": {
-          "type": "Integer",
-          "pk": true
-        },
-        "ent2_id": {
-          "type": "Integer",
-          "pk": true,
-          "fk": {
-            "to": { "entity": "Entity_2", "column": "ent2_id" },
-            "type": "many_to_one"
-          }
-        },
-        "ent1_description": {
-          "type": "Varchar"
-        }
-      }
-    }'>
+  <div class='mu-sql-table'
+    data-name='Personas'
+    data-columns='[{"name": "id_persona", "pk": true}, "Nombre", "Apellido", {"name": "Pareja", "fk": true}]'
+    data-rows='[
+      [1, "Homero", "Simpson", 2],
+      [2, "Marge", "Bouvier", 1],
+      [3, "Moe", "Szyslak", null]
+    ]'>
```

Comment:
I really like this format when describing a new table from scratch. 

However, if I already have a table, I think it is cumbersome to convert if to this format. 

Could we support an alternate and complementary format where the table is created using an standard `<table>`? 

Something like this: 

```
<table class='mu-sql-table' data-name='Personas'>
  <th>
    <td data-pk="true" >id_persona</td>
    <td>nombre</td>
    <td>apellido</td>
    <td data-fk="true">pareja</td>
  </th>
  <tr>
    <td>1</td>
    <td>"Homero"</td>
    <td>"Simpson"</td>
    <td>2</td>
  </tr>
  <tr>
    <td>2</td>
    <td>"Marge"</td>
    <td>"Bouvier"</td>
    <td>1</td>
  </tr>
  <tr>
    <td>3</td>
    <td>"Moe"</td>
    <td>"Szyslak"</td>
    <td>n</td>
  </tr>
</table>
```

Reply:
We need a header to put the Entity name.

if you write the table into a div work fine too:

```html
<div class="mu-sql-table">
  <header>Personas</header>
  <table>
    <thead>
      <tr>
        <th><i class="fa fa-fw fa-key mu-sql-table-pk"></i><span>id_persona</span></th>
        <th><span>Nombre</span></th>
        <th><span>Apellido</span></th>
        <th><i class="fa fa-fw fa-key mu-sql-table-fk"></i><span>Pareja</span></th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>1</td>
        <td>Homero</td>
        <td>Simpson</td>
        <td>2</td>
      </tr>
      <tr>
        <td>2</td>
        <td>Marge</td>
        <td>Bouvier</td>
        <td>1</td>
      </tr>
      <tr>
        <td>3</td>
        <td>Moe</td>
        <td>Szyslak</td>
        <td>NULL</td>
      </tr>
    </tbody>
  </table>
</div>

```

Reply:
Great! Then please also document this way of defining a table :smile:

Reply:
That HTML render this table

![image](https://user-images.githubusercontent.com/1039278/34625418-5a6bd56a-f237-11e7-8ebd-314465c8462b.png)